.github/workflows/push-event.yml

```
@@ -46,6 +84,8 @@ jobs:
           STORE_PASS: ${{ secrets.STORE_PASS }}
           ALIAS: ${{ secrets.ALIAS }}
           KEY_PASS: ${{ secrets.KEY_PASS }}
+          VERSION_CODE: ${{ env.VERSION_CODE }}
+          VERSION_NAME: ${{ env.VERSION_NAME }}
```

Following the suggestion
```suggestion
          VERSION_CODE: ${{ steps.android-version.outputs.VERSION_CODE }}
          VERSION_NAME: ${{ steps.android-version.outputs.VERSION_NAME }}
```